cookbooks/aws-parallelcluster-common/resources/partial/_get_package_version.rb

```diff
@@ -0,0 +1,9 @@
+def get_package_version(package_name)
+  cmd = "rpm -qi #{package_name} | grep Version | awk '{print $3}'"
```

Comment:
here we only have the rpm check, what about ubuntu?

Reply:
Same as above. Thank you!